finalPrep/README.md

```diff
@@ -18,37 +18,40 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://frontend.turing.io/today/))
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
 - `Add Week 1 Screenshot Here`
+<img width="1056" alt="Screen Shot 2021-08-07 at 9 11 08 AM" src="https://user-images.githubusercontent.com/81824035/128605017-767ec68b-29d7-4177-8069-0c220082f137.png">
```

Comment:
Your schedule has good bones, but could use some fleshing out. 
- Weekends are empty right now. I'd say I generally worked 8-10 hours on Turing stuff over the weekend, sometimes more, sometimes less. But do plan on weekends filling up some- that can be a key time to get project work done.
- As work piles up, I encourage you to change the name of some of your blocks to more focus what gets accomplished during that time. Switching tasks too frequently will not be productive. 
- I'd love to see some self-care practices added in... I see plenty of open time, but what are some routines or activities that will help you stay grounded?
- Finally, I strongly encourage a time set aside on weekends to honestly evaluate how you did keeping to your calendar the prior week, and make the adjustments that will help you be more successful in the week to come.